documentation/library_guide/index.rst

```diff
@@ -16,7 +19,7 @@ efforts across devices for high performance parallel applications.
    :maxdepth: 2
    :caption: Library Guide
 
-   overview
+   introduction
```

Comment:
Is the file renamed, or is a new file introduced? GitHub shows the diff as if a new file was added, yet the old file also remained intact. Let's make sure that there is only `introduction.rst` left when the patch is applied.

Reply:
Overview should have been renamed to Introduction. Good idea to check so only intro is left.